src/archive/zip/zip_test.go

```diff
@@ -464,6 +465,9 @@ func TestZip64LargeDirectory(t *testing.T) {
 	if testing.Short() {
 		t.Skip("skipping in short mode")
 	}
+	if runtime.GOARCH == "wasm" {
+		t.Skip("too slow on wasm")
+	}
```

Comment:
I imagine we might need to quantify this and potentially include some kind of TODO for re-enabling. What order of runtime are we talking about here?

Reply:
I just tested it again and it goes out of memory after a while.

Reply:
Can we add a comment explaining this here then?